packages/react/src/contents/hooks/useContentPage.ts

```diff
@@ -24,14 +24,20 @@ const useContentPage = <T = [ComponentType]>(
 ) => {
   const store = useStore();
 
-  const { enableAutoFetch = true, fetchConfig } = options;
+  const {
+    enableAutoFetch = true,
+    fetchConfig,
+    isCommercePage = false,
+  } = options;
 
   const query = useMemo(
     () => ({
-      contentTypeCode: ContentTypeCode.ContentPage,
-      codes: fetchQuery.slug.split('?')[0] as string,
+      contentTypeCode: isCommercePage
```

Comment:
Currently, to distinguish from the fetch to commerce pages endpoint directly we use the contentType commercePages and for the request to Serverless API, we use contentType contentPages, to generate the hash

Reply:
Ok, but in that case, why not use the `useCommercePages` hook by itself? I think that this signature is leaking implementation details that the consumer should not have to worry about. In my opinion, the correct solution would be for the `serverInitialState` to generate entries for both contentTypes in the store so the consumer does not need to worry about this.
What I mean is, the serverInitialState would set an entry for the `commercePages` and `contentPages` contentTypes containing the same data. I think the have the same data format so it would work. If that assumption is not correct, then we should adapt the responses from FO to the proper format.